shell/platform/darwin/ios/framework/Source/FlutterPlatformViewsTest.mm

```diff
@@ -15,9 +15,9 @@
 #import "flutter/shell/platform/darwin/ios/framework/Source/FlutterViewController_Internal.h"
 #import "flutter/shell/platform/darwin/ios/platform_view_ios.h"
 
-FLUTTER_ASSERT_NOT_ARC
+FLUTTER_ASSERT_ARC
 @class FlutterPlatformViewsTestMockPlatformView;
-static FlutterPlatformViewsTestMockPlatformView* gMockPlatformView = nil;
+__weak static FlutterPlatformViewsTestMockPlatformView* gMockPlatformView = nil;
```

Comment:
Ew :(

(Not the change, just the pattern itself of using a global static to track deallocation of an individual object; no changed need in this PR.)